src/npm-fastui/src/components/display.tsx

```diff
@@ -26,6 +28,28 @@ export const DisplayComp: FC<Display> = (props) => {
   }
 }
 
+// todo: this list should probably be defined in the models file
+const nestableSubcomponents = [
```

Comment:
I think there are two clean options here in terms of ways forward:
1. Something like #29 where we define a subset of components (or maybe they're not even included in regular components) that can be used in buttons and table cells and paragraphs (although I'm already seeing a problem that stuff like images belong in table cells but I think are illegal in paragraphs or buttons)
2. We allow anything in a table cell, and `DisplayObject` just becomes a component, which I've wanted otherwise anyway. Perhaps we have some logic for the default  components for table cells is a `DisplayObject`

Or maybe route 3 is we merge this to help @jimkring and worry about the ultimately correct solution later?